courses/free-courses-en.md

```diff
@@ -436,6 +436,7 @@
 * [Godot Wave Shooter Tutorials](https://www.youtube.com/playlist?list=PL6bQeQE-ybqAYoaWz_ZEE2X4wX6PhwCWR) - PlugWorld
 * [Make a Platform Shooter - GameMaker Studio 2](https://www.youtube.com/playlist?list=PL9FzW-m48fn3Ya8QUTsqU-SU6-UGEqhx6) - HeartBeast
 * [Multiplayer Shooter Tutorials - Godot](https://www.youtube.com/playlist?list=PL6bQeQE-ybqDmGuN7Nz4ZbTAqyCMyEHQa) - PlugWorld
+* [Unity 2D Game Development](https://www.youtube.com/watch?v=3Vpf6vFOXTQ&list=PLvUqRm2B9RRBgJipfDmFR7sFhEwBn7aGT) - Rabid Gremlin
```

Comment:
- For YT we prefer playlist urls
- Don't invent titles and creators. Use as provided by original source.

```suggestion
* [Run Bunny, Run! Creating a 2D game in Unity](https://www.youtube.com/playlist?list=PLvUqRm2B9RRBgJipfDmFR7sFhEwBn7aGT) - Rabidgremlin
```